File: threads/cpp/qt/threada.cpp
#include "threada.h"
#include <QDebug>

ThreadA::ThreadA(QObject *parent) :
	QThread(parent)
{

}

void ThreadA::run()
{
	for(int i=0; i<10; i++)
	{
		qDebug("A");
		sleep(1);
	}
}

File: threads/java/src/threads/ThreadA.java
package threads;

public class ThreadA extends Thread {
	
	@Override
	public void run() 
	{		
		for(int i=0; i<10; i++)
		{
			try {
				sleep(500);
		    }
		    catch(InterruptedException e) 
		    {
		    }
			 
			System.out.println("A");
			
		}
	}
}
